libethereum/Executive.cpp

```diff
@@ -160,17 +160,20 @@ string StandardTrace::json( bool _styled ) const {
         []( std::string a, Json::Value b ) { return a + Json::FastWriter().write( b ); } );
 }
 
-Executive::Executive( Block& _s, BlockChain const& _bc, unsigned _level )
+Executive::Executive( Block& _s, BlockChain const& _bc, const u256& _gasPrice, unsigned _level )
     : m_s( _s.mutableState() ),
       m_envInfo( _s.info(), _bc.lastBlockHashes(), 0 ),
       m_depth( _level ),
-      m_sealEngine( *_bc.sealEngine() ) {}
+      m_sealEngine( *_bc.sealEngine() ),
+      m_systemGasPrice( _gasPrice ) {}
 
-Executive::Executive( Block& _s, LastBlockHashesFace const& _lh, unsigned _level )
+Executive::Executive(
+    Block& _s, LastBlockHashesFace const& _lh, const u256& _gasPrice, unsigned _level )
     : m_s( _s.mutableState() ),
       m_envInfo( _s.info(), _lh, 0 ),
       m_depth( _level ),
-      m_sealEngine( *_s.sealEngine() ) {}
+      m_sealEngine( *_s.sealEngine() ),
+      m_systemGasPrice( _gasPrice ) {}
```

Comment:
You did not initialize m_baseGasRequired variable.

Reply:
1) It's not mine.
2) It's initialized in initialize()
3) I have separate task to "look through all uninitialized class members" ;)